cluster/storage/src/main/java/com/linkedin/openhouse/cluster/storage/hdfs/HdfsStorage.java

```diff
@@ -37,19 +36,4 @@ public StorageType.Type getType() {
   public StorageClient<?> getClient() {
     return hdfsStorageClient;
   }
-
-  /**
-   * Allocates Table Space for the HDFS storage.
-   *
-   * <p>tableLocation looks like: /{rootPrefix}/{databaseId}/{tableId}-{tableUUID} We strip the
-   * endpoint to ensure backward-compatibility. This override should be removed after resolving <a
-   * href="https://github.com/linkedin/openhouse/issues/121">
-   *
-   * @return the table location
-   */
-  @Override
-  public String allocateTableLocation(
```

Comment:
we ought to be careful with this. The reason we didn't make this change earlier was to ensure backward compatibilty. I would hold off checking this in until we keep the behavior compatible in [li-openhouse](https://github.com/linkedin-multiproduct/li-openhouse/blob/master/services/tables-extensions/src/main/java/com/linkedin/openhouse/cluster/storage/hdfs/LiHdfsStorage.java).  Could you make a change there to not call `super.allocateTableStorage`?